pkg/controllermanager/metrics/workqueue.go

```diff
@@ -32,16 +32,26 @@ var (
 		Help: "Count of item additions to a workqueue.",
 	}, workqueueLabels)
 
-	workqueueLatency = prometheus.NewSummaryVec(prometheus.SummaryOpts{
+	workqueueLatency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
 		Name: "garden_cm_workqueue_latency_milliseconds",
 		Help: "Time in milliseconds an item remains in the workqueue before it get processed.",
 	}, workqueueLabels)
 
-	workqueueDuration = prometheus.NewSummaryVec(prometheus.SummaryOpts{
+	deprecatedWorkqueueLatency = prometheus.NewSummaryVec(prometheus.SummaryOpts{
+		Name: "garden_cm_workqueue_latency_milliseconds_deprecated",
```

Comment:
why was the metric name changed - I would expect it to be `garden_cm_workqueue_duration_seconds`

Reply:
What do you mean? The deprecated metric is called `garden_cm_workqueue_latency_milliseconds_deprecated`, and the new metric is called `garden_cm_workqueue_duration_milliseconds`

Why "`seconds`"?

Reply:
See https://prometheus.io/docs/practices/naming/#metric-names only base units should be used - `seconds`, `meters`, `bytes` and so

Reply:
Fine, but the name did not follow these conventions previously. Please reach out to @dkistner regarding this. I just adapted the parts to the new 1.14 client-go metrics interface.